examples/doc_mappers/hdfs_doc_mapper.json

```diff
@@ -0,0 +1,40 @@
+{
+    "store_source": true,
```

Comment:
I think `"store_source": false` might be better here. (The json in the _source field can be confusing)

Reply:
done